doc/tools/make_rst.py

```diff
@@ -1438,5 +1439,30 @@ def sanitize_operator_name(dirty_name, state):  # type: (str, State) -> str
     return clear_name
 
 
+def indent_bullets(text):  # type: (str) -> str
+    # Take the text and check each line for a bullet point represented by "-", "+", or "*"
+    # Where found, indent the given line by a further "\t"
+    # Used to properly indent bullet points contained in the description for enum values
+    pos = 0
+    while True:
+        pos = text.find("\n", pos)
+        if pos == -1:
+            break
+
+        indent_level = 0
+        while text[pos + 1] == "\t":
+            pos += 1
+        pre_text = text[: pos + 1]
+        post_text = text[pos + 1 :]
+
+        if post_text.startswith("-") or post_text.startswith("+") or post_text.startswith("*"):
+            post_text = "\t" + post_text
+            pos += 1
+        pos += 1
+        text = pre_text + post_text
+
+    return text
```

Comment:
Here's an alternative version. I think it should work:
```suggestion
	bullet_points = "-+*"
	
	lines = text.splitlines(keepends=True)	
	for line_index, line in enumerate(lines):
		pos = 0
		while pos < len(line) and line[pos] == "\t":
			pos += 1		
		if pos < len(line) and line[pos] in bullet_points:
			lines[line_index] = line[:pos] + "\t" + line[pos:]
	return "".join(lines)
```